src/main/java/org/shanerx/tradeshop/enumys/Commands.java

```diff
@@ -34,49 +34,65 @@
 
 public enum Commands {
 
-    HELP(Lists.newArrayList("help", "?"), Permissions.HELP, 1, 2, false),
-    SETUP(Lists.newArrayList("setup", "start", "create", "make"), Permissions.HELP, 1, 1, false),
-	BUGS(Lists.newArrayList("bugs", "bug"), Permissions.NONE, 1, 1, false),
-    ADDMANAGER(Lists.newArrayList("addManager"), Permissions.NONE, 2, 2, true),
-    REMOVEMANGAER(Lists.newArrayList("removeManager", "delManager"), Permissions.NONE, 2, 2, true),
-    ADDMEMBER(Lists.newArrayList("addMember"), Permissions.NONE, 2, 2, true),
-    REMOVEMEMBER(Lists.newArrayList("removeMember", "delMember"), Permissions.NONE, 2, 2, true),
-    ADDPRODUCT(Lists.newArrayList("addProduct", "setProduct"), Permissions.NONE, 1, 3, true),
-    ADDCOST(Lists.newArrayList("addCost", "setCost"), Permissions.NONE, 1, 3, true),
-    OPEN(Lists.newArrayList("open"), Permissions.NONE, 1, 1, true),
-    CLOSE(Lists.newArrayList("close"), Permissions.NONE, 1, 1, true),
-    WHO(Lists.newArrayList("who"), Permissions.INFO, 1, 1, true),
-    WHAT(Lists.newArrayList("what", "peek", "windowShop", "shop", "view"), Permissions.INFO, 1, 1, true),
-    RELOAD(Lists.newArrayList("reload"), Permissions.ADMIN, 1, 1, false),
-    SWITCH(Lists.newArrayList("switch"), Permissions.EDIT, 1, 1, true);
+    HELP(Lists.newArrayList("help", "?"), Permissions.HELP, 1, 2, false, "Display help message", "/ts help [command]"),
```

Comment:
Can we pick either /ts or /tradeshop for these
Also we should put in a placeholder for the command name and just have it replace when it is retrieved that way we can use all the aliases  like you have for the later commands. This also leads to having support for servers changing the commands(for language or appearance purposes)
Last some of the commands are missing the /, i think it looks fine either way but we do need to pick a way

I can fix all these if you want just let me know

Reply:
You are right about having to be consistent, my mistake. I think displaying /tradeshop is better than its short version /ts.

I don‘t think placeholders are necessary, it would be useful for non English servers, but it would force us to bloat the source code by complicating stuff.

Reply:
the only difference would be a fairly simple replace in #usage and a small loop for building the string. I was planning on making the command names customizable by the server anyways, so this would just be a step that's probably going to be taken anyways.

and using the full name for command usage is usually what i default to

Reply:
We could do that. However, since that isn’t a priority I would prefer to add it in 2.1.0.

It would be a significant feature (despite its simple implementation) and we are already introducing major changes.

No point rushing it in now.

Reply:
kk im down with that

Reply:
I wasn't planning on fully implementing it yet, just was saying since we're making this now we might as well make it in a way that doesn't require use to change it right after we release 2.0